flipperdevices/flipperzero-firmware
# [FL-1064] iButton CLI commands

# What's new

Add iButton CLI commands:

- tm read
- tm write <key_type>  <key_data>
- tm emulate <key_type>  <key_data>

# Verification

- Build for f4 anf f5 targets
- Verify correct cli commads execution

# Checklist (do not modify)
